google_symptoms/delphi_google_symptoms/run.py

```diff
@@ -47,10 +59,29 @@ def run_module():
             df = df.loc[~df["val"].isnull(), :]
             df = df.reset_index()
             sensor_name = "_".join([smoother, "search"])
-            create_export_csv(
+            exported_csv_dates = create_export_csv(
                 df,
                 export_dir=export_dir,
                 start_date=SMOOTHERS_MAP[smoother][1](export_start_date),
                 metric=metric.lower(),
                 geo_res=geo_res,
                 sensor=sensor_name)
+            
+            if not exported_csv_dates.empty:
+                csv_export_count += exported_csv_dates.size
+                if not oldest_final_export_date:
+                    oldest_final_export_date = max(exported_csv_dates)
+                oldest_final_export_date = min(
+                    oldest_final_export_date, max(exported_csv_dates))
+
+    elapsed_time_in_seconds = round(time.time() - start_time, 2)
+    max_lag_in_days = None
+    formatted_oldest_final_export_date = None
+    if oldest_final_export_date:
+        max_lag_in_days = (datetime.now() - oldest_final_export_date).days
+        formatted_oldest_final_export_date = oldest_final_export_date.strftime("%Y-%m-%d")
+    logger.info("Completed indicator run",
+        elapsed_time_in_seconds = elapsed_time_in_seconds,
+        csv_export_count = csv_export_count,
+        max_lag_in_days = max_lag_in_days,
+        oldest_final_export_date = formatted_oldest_final_export_date)
```

Comment:
This whole block seems to be repeated frequently within and across indicators.  Is there a good way to make this a function or something else reusable?

Reply:
Definitely. The wrinkle is that some of the indicators use quite different setups (see e.g., change), so waiting to refactor after I've worked out how to handle those (it may be better to push this all the way into the csv_export function, for example)

I'm not too concerned with copypasting the boilerplate for now since it's just logging code.